### DISCUSSION TITLE
Default Backup Save Location on Win10?

### DISCUSSION THREAD

**Discussion by Blushock** [jendrikseipp/rednotebook#670] [Q&A]
Hi? On Windows 10, what is the default location RedNotebook saves backups in? I currently save them in a custom Desktop Folder, but could not find the default location or reset it.

**Comment by jendrikseipp** [comment]
The default backup directory is your home directory. After the first backup, the default backup directory becomes the one where you stored the last backup. You can see the backup directory in your home directory under `.rednotebook/configuration.cfg`.
